lib/jsonrpc/websocket.go

```diff
@@ -419,6 +422,32 @@ func (c *wsConn) handleFrame(ctx context.Context, frame frame) {
 	}
 }
 
+func (c *wsConn) closeInFlight() {
+	for id, req := range c.inflight {
+		req.ready <- clientResponse{
+			Jsonrpc: "2.0",
+			ID:      id,
+			Error: &respError{
+				Message: "handler: websocket connection closed",
```

Comment:
Should probably set a Code that isn't 0, it mildly bugs me.

Reply:
Yeah, agree, will choose `2` since would be the next unused value.